src/pages/workspaces/workspace/analysis/modals/compute-modal-components.ts

```diff
@@ -0,0 +1,109 @@
+// eslint-disable-next-line lodash-fp/use-fp
```

Comment:
I'm kind of wary about creating a generically named module that accumulate many unrelated components. What do you think about breaking this up into separate modules? Or naming it something more specific like "persistent disk controls"?